### PR TITLE
feat(graphql-api): implement security/devices resolvers

### LINKED ISSUE #5230: Implement query resolver for security/devices on settings page
The two sections (Security, Attached Devices) of the new Settings page should be implemented by calling into auth-server. These resolvers should then return that data.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1837)
┆Issue Number: FXA-1837

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* We want to show a users attached clients and security settings.

This commit:

* Retrieves the requested graph fields from the auth-server and
  resolves them.

Closes #5230

### CODE REVIEW

**Comment by lmorchard** [comment]
Starting to take a look

File: packages/fxa-graphql-api/.prettierrc [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
   "printWidth": 100,
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none",
```
Comment (lmorchard) [inline packages/fxa-graphql-api/.prettierrc:4]: We have `"trailingComma": "es5"` [in other `.prettierrc configs](https://github.com/mozilla/fxa/search?q=%22trailingComma%22%3A+%22es5%22&unscoped_q=%22trailingComma%22%3A+%22es5%22) - probably want that here too?

File: packages/fxa-graphql-api/src/lib/datasources/authServer.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+/**
+ * Converts an object with keys that are possibly snake_cased to an
+ * object with keys that are camelCased.
+ *
+ * @param obj Object with string keys
+ */
+export function snakeToCamelObject(obj: { [key: string]: any }) {
+  return Object.fromEntries(Object.entries(obj).map(([k, v]) => [snakeToCamel(k), v]));
```
Comment (lmorchard) [inline packages/fxa-graphql-api/src/lib/datasources/authServer.ts:23]: Not a real issue, probably not even a nit so much as a preference but: I kind of hate doing this sort of normalization. I've been bitten in the past by mistakes / confusion around properties from upstream data sources. I might be missing it - but is there a test or a test with a fixture that looks like auth-server API response to exercise this property renaming?

Comment (bbangert) [reply packages/fxa-graphql-api/src/lib/datasources/authServer.ts:23]: Threw a test in to verify the camelCase conversion.

Comment (bbangert) [reply packages/fxa-graphql-api/src/lib/datasources/authServer.ts:23]: The auth-server has a substantial amount of response validation to ensure it returns this correctly. I'm not sure adding more validation to make sure our response validation works helps a lot.

**Pull request merged**